README-offloaded-indexing.md

```diff
@@ -0,0 +1,79 @@
+### Cluster with remote indexing node, this a little manual since deploy vars are getting messy
+    $ export GIT_REL='v103rc2' && export ENCD_REL='v103rc2' && echo "$GIT_REL $ENCD_REL"
+
+    # Normal elasticsearch cluster
+    $ bin/deploy -b "$GIT_REL" --cluster-name "$ENCD_REL" --es-wait --dry-run
+        RC:   --release-candidate
+        Test: --test
+        Prod: --profile-name production --candidate
+
+    # Head frontend node: 12 app workers, 8 index workers
+    $ export ES_IP=172.31.25.24
+    $ bin/deploy -b $GIT_REL --es-ip $ES_IP --cluster-name "$ENCD_REL" --remote-indexing --primary-indexing --index-procs 8 --app-workers 12 --pg-open --do-batchupgrade no --dry-run
+        RC:   --release-candidate
+        Test: --test
+        Prod: --profile-name production --candidate
+
+    # Check environment vars
+    $ cat /etc/environment
+        * ENCD_APP_WORKERS=12
+        * ENCD_INDEX_PROCS=8
+        * ENCD_REMOTE_INDEXING=true
+
+    # ssh on and stop apache after the instance restart
+    $ sudo service apache2 stop
+
+    # Check encoded base.ini
+    [composite:indexer]
+    set queue_worker_processes = 8
+    set remote_indexing = true
+    set remote_indexing_threshold = 10001
+
+    # Check apache
+    App workers should be 12
+    indexer, vis, and region should exist
+
+    # Indexer node: 8 app workers, 16 index workers
+    $ export PG_IP=172.31.30.133
+    $ bin/deploy -b $GIT_REL --es-ip $ES_IP --pg-ip $PG_IP -n "$ENCD_REL-indexer" --cluster-name "$ENCD_REL" --no-indexing --primary-indexing --app-workers 8 --do-batchupgrade no --dry-run
```

Comment:
--no-indexing and --primary-indexing seem in opposition.    I think --primary-indexing overrides (from looking at deploy)...  probably no flags here are needed?
(also same comment about --do-batchupgrade)

Reply:
That could be true, but the vars are used for various parts of deployment.

I know these worked for me.  :)   And need people to deploy clusters with the commands we'll be using for v104 release.